docs/platform/dedicated-cpu/getting-started-with-dedicated-cpu/index.md

```diff
@@ -35,29 +37,35 @@ For more information on Dedicated CPU use cases, see our [Dedicated CPU Use Case
 
 1. Log in to the [Linode Cloud Manager](https://cloud.linode.com).
 
-2. Click on the **Create** dropdown menu at the top left of the page, and select "Linode".
+1. Click on the **Create** dropdown menu at the top left of the page, and select the **Linode** option.
 
-3. Select a [Distribution](/docs/quick-answers/linux/choosing-a-distribution/), [One-Click App](/docs/platform/one-click/how-to-use-one-click-apps-at-linode/), or [Image](/docs/platform/disk-images/linode-images/) to deploy from.
+1. Select a [Distribution](/docs/quick-answers/linux/choosing-a-distribution/), [One-Click App](/docs/platform/one-click/how-to-use-one-click-apps-at-linode/), or [Image](/docs/platform/disk-images/linode-images/) to deploy from.
```

Comment:
just changing the list numbering back to `1.` for each, because Hugo can automatically convert these into the right list number. If we ever need to insert new steps, this helps us avoid manually re-numbering later steps in the list. Though, maybe there's a conversation to be had here about if there are other reasons we'd want the actual list numbers in the markdown file?